tools/fw-update/readme.md

```diff
@@ -13,67 +13,73 @@ After installing `librealsense` run ` rs-fw-update -l` to launch the tool and pr
 An example for output for a D415 camera is:
 
 ```
-connected devices:
-1) Name: Intel RealSense D415, serial number: 725112060411, ASIC serial number: 012345678901, firmware version: 05.11.01.100, USB type: 3.2
+Connected devices:
+1) [USB] Intel RealSense D435IF s/n 038322070306, update serial number: 039223050231, firmware version: 5.15.1
 ```
 
-Then we will provid the serial number to identify the device together with the path to firmware file that we want to update ` rs-fw-update -s 725112060411 -f Signed_Image_UVC_5_11_6_250.bin`.
+Then we will provid the serial number to identify the device together with the path to firmware file that we want to update ` rs-fw-update -s 038322070306 -f Signed_Image_UVC_5_15_1_0.bin`.
 An example for the expected output is:
 
 ```
-search for device with serial number: 725112060411
+Search for device with serial number: 038322070306
 
-update to FW: Signed_Image_UVC_5_11_6_250.bin
+Updating device FW:
+[USB] Intel RealSense D435IF s/n 038322070306, update serial number: 039223050231, firmware version: 5.15.1
 
-updating device:
-Name: Intel RealSense D415, serial number: 725112060411, ASIC serial number: 012345678901, firmware version: 05.11.01.100, USB type: 3.2
+Firmware update started. Please don't disconnect device!
 
-firmware update started
+Firmware update progress: 0[%]
 
-firmware update progress: 100[%]
+Firmware update done
 
-firmware update done
+Waiting for device to reconnect...
 
-device 725112060411 successfully updated to FW: 05.11.06.250
+Device 038322070306 successfully updated to FW: 5.15.1
 ```
 
-In case only one camera is connected you can simply run ` rs-fw-update -f Signed_Image_UVC_5_11_6_250.bin`.
+In case only one camera is connected you can simply run ` rs-fw-update -f Signed_Image_UVC_5_15_1_0.bin`.
 
 A camera/s might be in a recovery state, in such case listing the devices will output the following:
 
 ```
-connected devices:
-1) Name: Intel RealSense D4xx Recovery, serial number: unknown, ASIC serial number: 012345678901, firmware version: unknown, USB type: unknown
+Connected devices:
+1) [0ADB] Intel RealSense D4XX Recovery, update serial number: 039223050231, firmware version: 5.16.0.1
 ```
 
-In such case we can use the recovery flag and run ` rs-fw-update -r -f Signed_Image_UVC_5_11_6_250.bin`
+In such case we can use the recovery flag and run ` rs-fw-update -r -f Signed_Image_UVC_5_15_1_0.bin`
 An example for the expected output is:
 
 ```
-update to FW: Signed_Image_UVC_5_11_6_250.bin
+Update to FW: C:\Users\nzini\Downloads\d400_series_fw_5_15_1_0\Signed_Image_UVC_5_15_1_0.bin
```

Comment:
This return the whole path?
I see nzini :)

Reply:
missed it, now it's fixed